### ISSUE TITLE
When x values come in as POSIX, the xlim should have the same timezone

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#285]
dates! :fire: 

Right now the xlim comes back w/ the user's timezone, even if the vals coming in to x have "UTC" or something else specified.

**Comment by jordansread** [comment]
version `‘0.6.0’` (same as in `‘0.5.4’`)

``` r
x.date <- seq(as.POSIXct("2000-01-01", tz = 'UTC'), by = "month", length.out = 10)
x.date
 [1] "2000-01-01 UTC" "2000-02-01 UTC" "2000-03-01 UTC" "2000-04-01 UTC" "2000-05-01 UTC" "2000-06-01 UTC" "2000-07-01 UTC" "2000-08-01 UTC" "2000-09-01 UTC" "2000-10-01 UTC"
gs <- gsplot() %>%
       points(x.date,1:10) %>%
       lines(6:14,6:14,side=c(3,4)) %>%
       grid(side=c(3,4))

xlim(gs)
$side.1
[1] "1999-12-31 18:00:00 CST" "2000-09-30 19:00:00 CDT"

$side.3
[1]  6 14
```

**Comment by lindsayplatt** [comment]
culprit: `strip_pts`
why: `out <- c()` and then `out <- append(out, v.vals)` where `v.vals` is the vector of dates. This makes v.vals a number and in the users timezone.

``` r
v.vals <- seq(as.POSIXct("2000-01-01", tz = 'UTC'), by = "month", length.out = 10)

out <- c()
append(out, v.vals)
 [1] "1999-12-31 18:00:00 CST" "2000-01-31 18:00:00 CST" "2000-02-29 18:00:00 CST" "2000-03-31 18:00:00 CST"
 [5] "2000-04-30 19:00:00 CDT" "2000-05-31 19:00:00 CDT" "2000-06-30 19:00:00 CDT" "2000-07-31 19:00:00 CDT"
 [9] "2000-08-31 19:00:00 CDT" "2000-09-30 19:00:00 CDT"

out <- c()
c(out, v.vals
[1] 946684800 949363200 951868800 954547200 957139200 959817600 962409600 965088000 967766400 970358400

# maybe do something like this: 
if(is.null(out)){
    out <- v.vals
} else {
    out <- append(out, v.vals)
}

out
[1] "2000-01-01 UTC" "2000-02-01 UTC" "2000-03-01 UTC" "2000-04-01 UTC" "2000-05-01 UTC" "2000-06-01 UTC"
 [7] "2000-07-01 UTC" "2000-08-01 UTC" "2000-09-01 UTC" "2000-10-01 UTC"
```

**Comment by lindsayplatt** [comment]
Created a function called `append_keepclass` (maybe _keepTZ would be more accurate), but still don't have a way to add NAs.

``` r
v.vals <- seq(as.POSIXct("2000-01-01", tz = 'UTC'), by = "month", length.out = 10)

 c(v.vals, NA)
 [1] "1999-12-31 18:00:00 CST" "2000-01-31 18:00:00 CST" "2000-02-29 18:00:00 CST" "2000-03-31 18:00:00 CST"
 [5] "2000-04-30 19:00:00 CDT" "2000-05-31 19:00:00 CDT" "2000-06-30 19:00:00 CDT" "2000-07-31 19:00:00 CDT"
 [9] "2000-08-31 19:00:00 CDT" "2000-09-30 19:00:00 CDT" NA       

append(v.vals, NA)
 [1] "1999-12-31 18:00:00 CST" "2000-01-31 18:00:00 CST" "2000-02-29 18:00:00 CST" "2000-03-31 18:00:00 CST"
 [5] "2000-04-30 19:00:00 CDT" "2000-05-31 19:00:00 CDT" "2000-06-30 19:00:00 CDT" "2000-07-31 19:00:00 CDT"
 [9] "2000-08-31 19:00:00 CDT" "2000-09-30 19:00:00 CDT" NA  
```

**Comment by lindsayplatt** [comment]
closed by #413
